solidity/contracts/converter/LiquidTokenConverter.sol

```diff
@@ -0,0 +1,278 @@
+pragma solidity 0.4.26;
+import './BancorConverter.sol';
+import './interfaces/ITypedConverterFactory.sol';
+
+/*
+    LiquidTokenConverter Factory
+*/
+contract LiquidTokenConverterFactory is ITypedConverterFactory {

```

Comment:
@yudilevi:
1. Why not in a separate file?
2. If not in a separate file, then why here and not in `ConverterFactory.sol`?
The incentives for #2 are:
1. Similar code (both contracts inherit from `ITypedConverterFactory`)
2. Short code (both contracts implement a single function `createConverter`)